puppetserver-standalone/Dockerfile

```diff
@@ -44,4 +44,14 @@ EXPOSE 8140
 ENTRYPOINT ["dumb-init", "/docker-entrypoint.sh"]
 CMD ["foreground" ]
 
+HEALTHCHECK --interval=10s --timeout=10s --retries=90 CMD \
+  curl --fail -H 'Accept: pson' \
+  --resolve 'puppet:8140:127.0.0.1' \
+  --cert   $(puppet config print hostcert) \
+  --key    $(puppet config print hostprivkey) \
+  --cacert $(puppet config print localcacert) \
+  https://puppet:8140/production/status/test \
```

Comment:
We probably should use the current official API (`https://puppet:8140/puppet/v3/status/test?environment=production`): https://docs.puppet.com/puppet/latest/http_api/http_status.html

Also, unfortunately this is only working if production is an actual environment (which may or may not be the case).